repo2docker/buildpacks/repo2docker-entrypoint

```diff
@@ -3,6 +3,7 @@
 # we use a login shell to get a fully configured shell environment
 # (e.g. sourcing /etc/profile.d, ~/.bashrc, and friends)
 if [[ ! -z "${R2D_ENTRYPOINT:-}" ]]; then
+    chmod u+x "$R2D_ENTRYPOINT"
```

Comment:
To avoid a corner case where a file is owned by someone else but is already group/other executable how about only running `chmod` if it's not already executable by testing with something like `[ -x "$R2D_ENTRYPOINT" ]`?

Reply:
Good idea - I have added this to the latest commit.